docs/audit_report/changes/topics/tls.yml

```
@@ -2,6 +2,8 @@ title: TLS Improvements and Extensions
 
 description: |
   Most notably, this adds support for user-provided Preshared Keys in TLS 1.3.
+  The second mayor extension is the support of TLS 1.3 with Hybrid PQ/T key
+  establishment.
```

I believe the support for PQ/T is more relevant than the addition of PSK. So, I suggest:

```suggestion
  Most notably, this adds support for quantum-secure hybrid PQ/T key
  establishment combining Kyber and classical elliptic curve algorithms. Also, support for
  user-provided Preshared Keys in TLS 1.3 is added.
```

---

Agreed!